TECHMANIA/Assets/Scripts/Locale.cs

```diff
@@ -165,6 +165,10 @@ public static string GetString(string key)
         params object[] args)
     {
         string format = GetString(formatKey);
-        return string.Format(format, args);
+#if UNITY_IOS
```

Comment:
`Paths.ReplacePlatformPathText` already is platform-specific, there's no need to be platform-specific at call sites.